### ISSUE TITLE
Packages from local dir

### ISSUE THREAD

**Issue by w41l** [rg3/slackroll#17]
A question regarding this:
For slackroll to be able to work with a third-party repository, the repository needs to provide a FILELIST.TXT file in the repository root, and packages need to be signed with external .asc files like the official ones.

In slackpkg+ I can add my own testing repo dir without FILELIST.TXT, x.asc and such.
```
# Local packages (you do not need metadata nor 'slackpkg update' command):
MIRRORPLUS['wlskde5']=dir://mnt/data/ftp/kde/
```

Can I do something like that with slackroll?

Thank you

**Comment by rg3** [comment]
No, you can't. Using a local repository should be fine with the file:// protocol, but you still need to generate the FILELIST.TXT file and package signature files. Not that it's hard to do.

FILELIST.TXT can be generated with something like:
`find . -print0 | xargs -0r ls -ld --time-style long-iso`

And package signatures can be trivially generated with a gpg key and:
`gpg -a -b -u USERHASH -o FILE.asc FILE`

**Comment by aclemons** [comment]
Closing stale issues.
